alstof-jeremy/README.md

```diff
@@ -0,0 +1,28 @@
+# Project Name
+
+**Author**: Alex Stoforandov and Jeremy Beck
+**Version**: 1.3.0
+
+## Overview
+
+The app is a simple blog engine built upon JQuery and SMACSS libraries.
+
+## Getting Started
+
+Clone the repo. Open index.html in your browser.
+
+## Architecture
+
+The app is built using HTML, CSS, Javascript, JQuery library and SMACSS methodolody.
+
+## Change Log
```

Comment:
make sure you are previewing your MD files. These are all jumbled together. If you make it a list, it'll look better, just add `- ` in front of each line.